tests/test_pandas_dataset.py

```diff
@@ -29,77 +31,12 @@ def test_expect_column_to_exist(self):
                 self.assertEqual(out['raised_exception'], True)
                 self.assertIn(t['error']['traceback_substring'], out['exception_traceback'])
 
-    def test_expect_table_row_count_to_be_between(self):
```

Comment:
Did we lose some tests here?

Reply:
This method was immediately overriden by the next, I suspect during the (partial) conversion to the JSON-style tests (see below). I didn't do that conversion, but I suspect it covered the same code. We'll need to revisit when we standardize on the testing style.